include/deal.II/lac/solver_cg.h

```diff
@@ -166,6 +182,33 @@ class SolverCG : public Solver<VECTOR>
          const VECTOR         &b,
          const PRECONDITIONER &precondition);
 
+  /**
+   * Connect a slot to retrieve the CG coefficients. The slot will be called
+   * with alpha as the first argument and with beta as the second argument.
+   * Called once per iteration.
```

Comment:
For example by pointing to a place where a pseudo-algorithm is listed that uses these names.